pkg/services/rocketchat/rocketchat_config.go

```diff
@@ -46,7 +47,9 @@ func (config *Config) SetURL(serviceURL *url.URL) error {
 	config.TokenA = path[1]
 	config.TokenB = path[2]
 	if len(path) > 3 {
-		if path[3][0:1] != "@" {
+		if serviceURL.Fragment != "" {
+			config.Channel = "#" + strings.TrimLeft(serviceURL.Fragment, "#")
```

Comment:
https://golang.org/pkg/strings/#TrimLeft

> To remove a prefix, use TrimPrefix instead.

No biggie here as the behavior will be the same, but to keep intentions clear, I think we should switch to `TrimPrefix` anyway.

Reply:
Implemented the suggested change